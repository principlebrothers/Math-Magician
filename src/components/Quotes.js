import React from 'react';
import './style/Quotes.css';

function Quotes() {
  return (
    <div className="quotes">
      <div className="quote">
        “Mathematics is not about numbers, equations, computations, or
        algorithms: it is about understanding.”
      </div>
      <cite className="quoteAttribute">&quot;William Paul Thurston&quot;</cite>
    </div>
  );
}

export default Quotes;
