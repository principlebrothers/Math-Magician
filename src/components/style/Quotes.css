.quotes {
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  margin: 40vh 0;
  font-size: 24px;
}
