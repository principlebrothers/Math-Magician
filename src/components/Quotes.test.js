import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from './Quotes';

test('Correctly render the Quotes componenet', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
