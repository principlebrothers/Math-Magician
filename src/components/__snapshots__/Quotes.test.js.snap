// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly render the Quotes componenet 1`] = `
<div
  className="quotes"
>
  <div
    className="quote"
  >
    “Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.”
  </div>
  <cite
    className="quoteAttribute"
  >
    "William Paul Thurston"
  </cite>
</div>
`;
